src/pages/home/sidebar/SidebarScreen.js

```diff
@@ -160,7 +162,7 @@ class SidebarScreen extends Component {
                                     text: this.props.translate('sidebarScreen.newGroup'),
                                     onSelected: () => Navigation.navigate(ROUTES.NEW_GROUP),
                                 },
-                                ...(Permissions.canUsePolicyRooms(this.props.betas) ? [
+                                ...(Permissions.canUsePolicyRooms(this.props.betas) && workspaces.length > 0 ? [
```

Comment:
You can do just `workspaces.length` here I believe.

Reply:
Updated